src/classes/route.py

```diff
@@ -46,6 +46,45 @@ def route_position_to_world_position(route: Route, position: float):
     
     return [world_x, world_y]
 
+def direction_at_route_position(route, position):
```

Comment:
Great job man! It looks and works great. Small thing, we're trying to add typing for all functions parameters and returns.